frontend/packages/client/src/pages/UserProfile.tsx

```diff
@@ -0,0 +1,132 @@
+import Blockies from 'react-blockies';
+import { Link as RouterLink } from 'react-router-dom';
+import { useWebContext } from 'contexts/Web3';
+import {
+  CommunityLinks,
+  CustomTab,
+  ShareProfileDropdown,
+  WalletAddress,
+} from 'components';
+import { useMediaQuery, useQueryParams } from 'hooks';
+import {
+  Box,
+  Button,
+  Flex,
+  HStack,
+  Link,
+  Spacer,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  VStack,
+} from '@chakra-ui/react';
+import PageContainer from 'layout/PageContainer';
+
+const UserProfile: React.FC = () => {
+  const {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
if you use `// @ts-expect-error: Enter reason here` then you don't need the eslint disable and we will have a documented reason why for why we need to ignore the line